### PR TITLE
Add rollback pipeline

### PR DESCRIPTION
**Pull request by mattgauntseo-sentry**
This PR causes pipedreams to include a rollback pipeline that is configured like so:

1. Rollback pipeline has an upstream dependency on the last pipeline in a pipedream (this means we only rollback to deploys that have been deployed to all regions)
2. When run, it'll pause and cancel all pipelines in pipedream
3. It'll get the SHA for the selected run of the last pipeline in the pipedream
4. Run the configured rollback stage for all pipelines in pipedream using the sha to select which pipeline to re-run.

### CODE REVIEW

File: v1.0.0/bash/rollback.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <PIPELINE_FLAGS>
+
+# Get sha from the given pipeline run to deploy to all pipedream pipelines.
+sha=$(gocd-sha-for-pipeline --material-name="${ROLLBACK_MATERIAL_NAME}")
+
+gocd-emergency-deploy \
+  --commit-sha="${sha}" \
+  --deploy-stage="${ROLLBACK_STAGE}" \
+  --pause-message="This pipeline was rolled back, please check with team before un-pausing." \
+  <PIPELINE_FLAGS>
```
Comment (bukzor-sentryio) [inline v1.0.0/bash/rollback.sh:15]:
```suggestion
  "$@"
```

Comment (bukzor-sentryio) [inline v1.0.0/bash/rollback.sh:15]: another option, would be to put the arguments in an environment variable named "ARGS", and add a snippet like

```sh
#!/bin/bash
if [[ "${ARGS:-}" ]]; then
  # shellcheck disable=SC2086
  set -- $ARGS   # note: no quoting, for word expansion
fi

...
"$@"
```

Comment (mattgauntseo-sentry) [reply v1.0.0/bash/rollback.sh]: Went with the other option below.

Comment (mattgauntseo-sentry) [reply v1.0.0/bash/rollback.sh]: Love this approach - testings locally running the script with set -x and output looks correct.

**Review by joshuarli — APPROVED** [review]
looks great!

**Pull request merged**